docs/operations-manual.md

```diff
@@ -8,15 +8,13 @@ For information regarding building MetaDIG software, please refer to the MetaDIG
 
 ## MetaDIG Service Dependencies
 
-The MetaDIG Assessment services are dependant on the following components which must be configured and installed before any of the services can be installed.
+The MetaDIG Assessment services are dependent on the following components which must be configured and installed before any of the services can be installed.
 
 ### Metadig Ingress
 
 A k8s ingress resource provides network traffic routing from outside the k8s cluster network to a k8s service.
 
-The metadig-engine ingress definition can be found [here](https://github.com/NCEAS/metadig-engine/blob/main/k8s/ingress-metadig.yaml). This k8s resource only needs to be created once, and at this time is
-not included in the Helm install, but could be if desired. Note that every time the ingress resource is deleted and recreated, as would happen with a 'helm delete' and 'helm install',
-the cert-manager sends a new request to the Let's Encrypt service to the Let's Encrypt service. Cert-manager is described in the DataONE k8s-cluster documentation [here](https://github.com/DataONEorg/k8s-cluster/blob/main/authentication/LetsEncrypt.md).
+The metadig-engine ingress definition can be found [here](https://github.com/NCEAS/metadig-engine/blob/main/k8s/ingress-metadig.yaml). This k8s resource only needs to be created once, and at this time is not included in the Helm install, but could be if desired. Note that every time the ingress resource is deleted and recreated, as would happen with a 'helm delete' and 'helm install', the cert-manager sends a new request to the Let's Encrypt service to the Let's Encrypt service. Cert-manager is described in the DataONE k8s-cluster documentation [here](https://github.com/DataONEorg/k8s-cluster/blob/main/authentication/LetsEncrypt.md).
```

Comment:
Does the LE request also happen with `helm upgrade`? We should generally be using `helm upgrade` for cleaner upgrades with no downtime.

Reply:
I'm not sure - what are the implications for the LE request? I run `helm upgrade` when I upgrade on production but have done `helm delete`/ `helm install` on the dev cluster before